src/main/groovy/life/qbic/datamodel/datasets/datastructure/files/nanopore/FastQFile.groovy

```diff
@@ -0,0 +1,30 @@
+package life.qbic.datamodel.datasets.datastructure.files.nanopore
+
+import life.qbic.datamodel.datasets.datastructure.files.DataFile
+
+/**
+ * <add class description here>
+ *
+ * @author: Sven Fillinger
+ */
+class FastQFile extends DataFile {
+
+    final private static String FILE_TYPE = "fastq"
+
+    final private static String NAME_SCHEMA = /.*\.fastq$/
+
+    protected FastQFile(String name, String relativePath, String fileType) {
+        super(name, relativePath, fileType)
+        validateName()
+    }
+
+    static FastQFile create(String name, String relativePath) {
```

Comment:
Same issue with the file type. It is not checked.

Reply:
The file type is checked in the super class DataFile